labrad/units.py

```diff
@@ -346,7 +348,7 @@ def __hash__(self):
         of messed up: 1001*m != 1.001*km because the latter is not an
         exact number.  C'est la floating point.
         '''
-        return hash(self.inBaseUnits()._value)
+        return hash(self[self.unit.base_unit])
```

Comment:
Since I'm trying to get up to speed with this, I'm going to ask a bunch of naive questions. Why do we have a hash function at all in this, and why does the docstring say something about converting from one SI prefix to another. Does hash not mean what I think it means?